### PR TITLE
Replace the URL's as they no longer working

### PR DESCRIPTION
**Pull request by dev192**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -39,9 +38,9 @@ class Home extends StatefulWidget {
 class _HomeState extends State<Home> {
   final String url =
       'https://corona-virus-stats.herokuapp.com/api/v1/cases/general-stats';
+  //need's to replace the URL's as they no longer working
   final String mostEffectedCountriesurl =
```
Comment (DeekshithRajBasa) [inline lib/main.dart:42]: If you find any new api please contribute to this and raise a PR

File: lib/main.dart [hunk 2]
```diff
@@ -71,13 +67,12 @@ class _HomeState extends State<Home> {
       });
     }
   }
-
-  int _selectedIndex =0;
-
+  
+  int _selectedIndex = 0;
+  //we can also do it with var _selectedIndex = 0;
```
Comment (DeekshithRajBasa) [inline lib/main.dart:72]: cool, we can also do this even!

File: lib/main.dart [hunk 3]
```diff
@@ -223,7 +215,7 @@ class mainScreen extends StatelessWidget {
                         crossAxisAlignment: CrossAxisAlignment.start,
                         children: <Widget>[
                           Text(
-                            'Covid-19 Tracker',
+                            'Covid19 Tracker',
```
Comment (DeekshithRajBasa) [inline lib/main.dart:218]: Does not make sense 😉

**Review by DeekshithRajBasa — APPROVED** [review]

File: lib/main.dart (hunk 2 above)
Comment (dev192) [reply lib/main.dart:72]: Yep, thanks I'll update this in my local

File: lib/main.dart (hunk 3 above)
Comment (dev192) [reply lib/main.dart:218]: As per my grammar, I don't use it

**Pull request merged**